components/server/src/prometheus-metrics.ts

```diff
@@ -130,11 +130,11 @@ export function increaseApiConnectionClosedCounter() {
 const apiCallCounter = new prometheusClient.Counter({
     name: "gitpod_server_api_calls_total",
     help: "Total amount of API calls per method",
-    labelNames: ["method", "statusCode"],
+    labelNames: ["method", "statusCode", "httpVersion"],
```

Comment:
This is presently wrong and needs fixing:
![image](https://github.com/user-attachments/assets/050f5575-cff3-4da2-a5f2-d60fd4bfabb8)

Reply:
Should be fixed in fedc28f8dfbf9f9d7a37ee11eb846050f533a8b8, test again before resolving